common/archiver/README.md

```diff
@@ -45,13 +46,22 @@ type HistoryArchiver interface {
 
 ```go
 type VisibilityArchiver interface {
+    // Archive is used to archive one workflow visibility record. Check the Archive() method of the HistoryArchiver interface for parameters' meaning and requirements for this method. 
+    // The only difference is that the ArchiveOption parameter won't include an option for recording process. 
+    // Please make sure your implementation is lossless. If any in-memory batching mechanism is used, then those batched records will be lost during server restarts. 
```

Comment:
Does it mean that the implementation should never do batching?

Reply:
It depends on how the batching is implemented. If something like our indexer is implemented, then it's fine. So I think the keyword here is in-memory.